<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpgradeOverStorage</class>
 <widget class="QDialog" name="UpgradeOverStorage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Storage full</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#UpgradeDialog
{
	background-color: #FBFBFB;
}

#wHeader
{
  background-color: white;
}

#bRocket, #bStorageFull
{
 border: none;
}

#lEncourageUpgrade
{
	color: #F0373A;
	font-family: Lato;
	font-size: 14px;
	font-weight: 600;
	line-height: 17px;
}

#lUsageStorage
{
	color: #333333;
	font-family: Lato;
	font-size: 14px;
	line-height: 24px;
}

#lAchievements
{
	color: #333333;
	font-family: &quot;Source Sans Pro&quot;;
	font-size: 14px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wHeader" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>192</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>192</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>35</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bStorageFull">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/images/storage_full.png</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lUsageStorage">
           <property name="text">
            <string>Your account is full ([A] / [B]). All file uploads to MEGA are currently disabled.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lEncourageUpgrade">
           <property name="text">
            <string>Please upgrade to PRO to enjoy more storage space (your uploads will automatically resume).</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wPlans" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>280</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>280</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wAchievements" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>68</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>68</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>-1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bRocket">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/achievements.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lAchievements">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Earn more storage quota with [A]Achievements[/A].</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
